event/powerlevels.go

```diff
@@ -32,6 +34,13 @@ type PowerLevelsEventContent struct {
 	HistoricalPtr *int `json:"historical,omitempty"`
 }
 
+// Notifications represents the content of the notifications key in an
+// m.room.power_levels state event content.
+// https://spec.matrix.org/v1.2/client-server-api/#mroompower_levels
+type PowerLevelsEventContentNotifications struct {
+	Room int `json:"room,omitempty"`
```

Comment:
Should probably do `RoomPtr` similar to `InvitePtr` and others so that it can have a getter that defaults to 50 when omitted